mms/utils/validators/validate_messages.py

```diff
@@ -35,10 +35,13 @@ def validate_load_message(msg):
         :return:
         """
         if u'modelPath' not in msg:
```

Comment:
unicode still required??? builtins should handle it

Reply:
This is used everywhere. So I'm leaving it there for now. 
Changing the unicode extension can be a part of a separate refactoring of this file.